session/src/test/java/uk/gov/di/ipv/cri/common/api/service/SignedJWTBuilder.java

```diff
@@ -173,6 +178,10 @@ SignedJWT build() {
             return signedJWT;
         } catch (JOSEException e) {
             throw new IllegalStateException(e);
+        } catch (JsonMappingException e) {
```

Comment:
Instead of 2 distinct catch blocks, you can condense this into one i.e. `} catch (JsonMappingException | JsonProcessingException e) {`

Reply:
We should also be careful, that PII is not going to be exposed in this exception